components/actiongroup/index.css

```diff
@@ -79,12 +78,20 @@ governing permissions and limitations under the License.
       border-radius: 0;
       z-index: 0;
 
+      &:after {
+        --spectrum-actionbutton-focus-ring-border-radius-adjusted: 0;
+        test: var(--spectrum-actionbutton-focus-ring-border-radius-adjusted);
```

Comment:
Looks like some leftover props slipped in.

Reply:
Oh, I wanted to ask you about this, I forgot I left it in, but this was my hack to get the unused vars to not clear out my new definition.